gettsim/transfers/kinderzuschlag/kinderzuschlag_eink.py

```diff
@@ -134,7 +134,7 @@ def anz_kinder_anspruch_per_hh(
     return kindergeld_anspruch.groupby(hh_id).transform("sum")
 
 
-def kinderzuschlag_eink_max(
```

Comment:
I am confused by name and docstring here. What exactly does the function return? Some income measure + the maximal child benefit surely is not the "maximal claim of child benefit" ?

Reply:
You are rightly confused.

This is to determine the maximum income amount for which kiz is paid (before 2019). And it depends on the kiz itself. So the function name was not so bad, but the docstring was.

Reply:
Cool, that makes sense!